embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/DataSourceModule.kt

```diff
@@ -69,6 +71,19 @@ internal class DataSourceModuleImpl(
         )
     }
 
+    override val pushNotificationDataSource: DataSourceState<PushNotificationDataSource> by dataSourceState {
+        DataSourceState(
```

Comment:
How do users enable this feature in their apps? I'm a little surprised that there isn't any parameter supplied for `configGate` although if that's existing behavior that's ok

Reply:
This is used by manually calling our public api, and automacally if the feature is enabled with this guide: https://embrace.io/docs/android/features/push-notifications/